lib/Task.js

```diff
@@ -116,6 +131,12 @@ class Task {
 
     return error
   }
+
+  handleError(msg, isCli) {
```

Comment:
I'm not sure. too. But our test uses `this.error` property, so we need to update `this.error` before calling `helpers.err()`